media/css/firefox/personal.less

```diff
@@ -0,0 +1,194 @@
+@import "../sandstone/lib.less";
+
+@font-face {
+    font-family: 'FiraSans Regular';
+    src: url('/media/fonts/FiraSans-Regular.woff') format('woff');
```

Comment:
I believe browsers shouldn't ("shouldn't" being the key word here; we should test to verify) download font files if the given page doesn't actually use them, so in theory just declaring a bunch of variants in CSS won't impact performance unless you use a lot of them on one page.

But that matter aside, I'd still vote for adding the other formats (TTF and EOT) to bedrock anyway since we know we'll need them in the future.